feature_layers/display-clusters/src/main/java/com/esri/samples/display_clusters/DisplayClustersSample.java

```diff
@@ -0,0 +1,171 @@
+public class DisplayClustersSample extends Application {
+
+  private ArcGISMap map;
+  private MapView mapView;
+  private PortalItem portalItem; // keep loadable in scope to avoid garbage collection
+  private FeatureLayer powerPlantsLayer;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Display Points Using Clustering Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a button to toggle clustering
+      Button featureClusteringToggle = new Button("Toggle Feature Clustering");
+      featureClusteringToggle.setDisable(true);
+
+      // create a portal and portal item, using the portal and item ID
+      var portal = new Portal("https://www.arcgis.com/");
+      portalItem = new PortalItem(portal, "8916d50c44c746c1aafae001552bad23");
+
+      map = new ArcGISMap(portalItem);
+      map.addDoneLoadingListener(() -> {
+        if (map.getLoadStatus() == LoadStatus.LOADED) {
+          featureClusteringToggle.setDisable(false);
+
+          powerPlantsLayer = (FeatureLayer)map.getOperationalLayers().get(0);
+
+          // when the button is clicked, toggle the feature clustering
+          mapView.setOnMouseClicked(mouseEvent -> {
+            if (mapView == null) {
+              return;
+            }
+
+            mapView.getCallout().setVisible(false);
+            Point2D point = new Point2D(mouseEvent.getX(), mouseEvent.getY());
+            ListenableFuture<IdentifyLayerResult> identifiedLayerResults = mapView.identifyLayerAsync(powerPlantsLayer, point, 3.0, false);
+            identifiedLayerResults.addDoneListener(() -> {
+              try {
+                if (identifiedLayerResults.get() == null || identifiedLayerResults.get().getPopups().isEmpty()) {
+                  return;
+                }
+
+                Popup popup = identifiedLayerResults.get().getPopups().get(0);
+
+                Callout callout = mapView.getCallout();
+                String htmlText = popup.getDescription();
+
+                WebView webView = new WebView();
+                VBox region = new VBox();
+                WebEngine webEngine;
+                webEngine = webView.getEngine();
+                webEngine.loadContent(htmlText);
+                webView.setMaxHeight(100);
+                webView.setMaxWidth(150);
+                region.getChildren().add(webView);
+
+                callout.setCustomView(region);
+
+                callout.screenToLocal(point);
+                callout.setVisible(true);
```

Comment:
I think `showCalloutAt` makes it visible so don't need to explicitly make it visible.